tests/test_sms.py

```diff
@@ -40,3 +39,18 @@ def test_red_sms_provider(requests_mock, timestamp_mock):
             "from": sender,
         }
     )
+
+
+@mock.patch("timeless.sms.requests")
```

Comment:
@vladarefiev I have to say, I hate this testing/mocking framework. I have no idea where/how/when ``RetrySendSMS`` is injected. It's a test for that class, apparently, but there's not a single ``RetrySendSMS`` in the test's code :D 

Is it possible to write the tests in such a way that they don't look like black magic?

Reply:
@amihaiemil No, this test is not for `RetrySendSMS` class.

`RetrySendSMS` - is  a decorator which implements by class (we can do it with function if you wish).
decorator decorate `send` method as you  can see. 
According to test. Here is should be integration with another service. So, we mock request.post function which makes request to external API and check own internal logic.
How would you see this PR, pls explain explicit and I try to implement it.

Reply:
@vladarefiev 
>  How would you see this PR, pls explain explicit and I'll try to implement it.

It should be fine. Now I see that ``RetrySendSMS`` is actually used as an annotation over method ``send()`` from  ``RedSMS``.

Maybe there's a term confusion. By ``decorator``, I mean the decorator pattern, which should look like this:

```
SMS sms = new RetrySendSMS(
    new RedSMS(...), 3
)
```

But I also like the annotation idea, if that's the Python way to go, then we leave it like this.

Reply:
@amihaiemil Got your idea. You are right. Yes, it's a little confusing pattern decorator and python's decorator